openxc-ios-framework/VehicleManager.swift

```diff
@@ -237,6 +261,17 @@ open class VehicleManager: NSObject, CBCentralManagerDelegate, CBPeripheralDeleg
       return
     }
     
+    // if the found VI list is empty, just return
```

Comment:
not required after this
 // if the name is given, look it up. Otherwise use any peripheral
      openXCPeripheral = (name != nil ? foundOpenXCPeripherals[name!] : foundOpenXCPeripherals.first?.1)